src/connectors/aws_account.py

```diff
@@ -0,0 +1,100 @@
+"""AWS Account List
+Lists your AWS Accounts
+"""
+
+from runners.helpers import db
+from runners.helpers.dbconfig import ROLE as SA_ROLE
+
+import boto3
+import datetime
+
+CONNECTION_OPTIONS = [
+    {
+        'name': 'source_role_arn',
+        'title': "source role arn",
+        'prompt': "assumer role arn",
+        'type': 'str',
+        'required': True
+    },
+    {
+        'name': 'destination_role_arn',
+        'title': "destination role arn",
+        'prompt': "destination role arn",
+        'type': 'str',
+        'required': True
+    },
+    {
+        'name': 'destination_role_external_id',
+        'title': 'destination_role_external_id',
+        'prompt': 'destination_role_external_id',
+        'type': 'str',
+        'required': True
+    }
+]
+
+LANDING_TABLE_COLUMNS = [
+    ('raw', 'VARIANT'),
+    ('event_time', 'TIMESTAMP_LTZ'),
+]
+
+
+def get_org_client(session_name, src_role_arn, dest_role_arn, dest_external_id):
```

Comment:
this seems like it's doing something else until the very end -- can we move out the `.client('organizations')` and give this helper method a better name? it seems like something that might be more general than this connector, also? what's a good place to put it?

Reply:
resolved